datafusion/common/src/error.rs

```diff
@@ -478,9 +478,16 @@ macro_rules! with_dollar_sign {
 ///     plan_err!("Error {val}")
 ///     plan_err!("Error {val:?}")
 macro_rules! make_error {
-    ($NAME:ident, $ERR:ident) => {
+    ($NAME:ident, $NAME_RAW: ident, $ERR:ident) => {
```

Comment:
Can you please document in comments what `$NAME` and `$NAME_RAW` are and how they are different?

my reading is that the `raw` is the name of an internal macro that gets generated. I think it would help to document more clearly what the difference in `plan_err!` and `plan_err_raw!` did (specifically that `plan_err` returns an `Err(plan_err_raw)`